options.go

```diff
@@ -1,68 +1,60 @@
 package goload
 
 import (
-	"context"
+	"github.com/HenriBeck/goload/pacer"
 	"time"
 )
 
-type LoadTestOptions struct {
-	Endpoints []Endpoint
-	// A config which allows to set the requests per minute for each registered endpoint at a central place
-	//
-	// This overrides any local endpoint setting where a default RPM value was set
-	RequestPerMinutePerEndpoint map[string]int32
-
-	DefaultEndpointTimeout time.Duration
-
-	LoadTestDuration time.Duration
-
-	RPMStrategy RPMStrategy
-
-	ContextModifiers []func(ctx context.Context) context.Context
+func WithDuration(loadTestDuration time.Duration) LoadTestOption {
+	return func(options *LoadTestOptions) {
+		options.duration = loadTestDuration
+	}
 }
 
-type LoadTestConfig func(options *LoadTestOptions)
-
-func WithEndpoints(endpoints ...Endpoint) LoadTestConfig {
+func WithExecutors(executors ...Executor) LoadTestOption {
 	return func(options *LoadTestOptions) {
-		options.Endpoints = append(options.Endpoints, endpoints...)
+		options.executors = append(options.executors, executors...)
 	}
 }
 
-func WithRequestsPerMinuteForEndpoints(data map[string]int32) LoadTestConfig {
+func WithLinearRampUpPacer(startRate, endRate pacer.Rate, rampUpDuration time.Duration) LoadTestOption {
 	return func(options *LoadTestOptions) {
-		options.RequestPerMinutePerEndpoint = data
+		options.pacer = pacer.NewLinearRampUpPacer(startRate, endRate, rampUpDuration)
 	}
 }
 
-func WithContextModifier(fn func(ctx context.Context) context.Context) LoadTestConfig {
+func WithConstantPacer(rate pacer.Rate) LoadTestOption {
 	return func(options *LoadTestOptions) {
-		options.ContextModifiers = append(options.ContextModifiers, fn)
+		options.pacer = pacer.NewConstantPacer(rate)
 	}
 }
 
-func WithDefaultEndpointTimeout(timeout time.Duration) LoadTestConfig {
```

Comment:
Any reason we removed the default endpoint timeout config?

Reply:
Missed it somehow 🤷‍♂️ it's back now